.gflows/config.yml

```diff
@@ -9,5 +9,5 @@ templates:
       - libs
       - libs/jobs
     dependencies:
-      - https://raw.githubusercontent.com/CoverGo/ci-workflow-libraries/v2.2/.gflows
+      - https://raw.githubusercontent.com/CoverGo/ci-workflow-libraries/be-528-sign-docker-image/.gflows
```

Comment:
The version will be updated when the following PR is merged to master - https://github.com/CoverGo/ci-workflow-libraries/pull/1.

Reply:
merged, let's use v2.3 instead of a branch

Reply:
Done